packages/upload-api/test/access-client-agent.js

```diff
@@ -347,17 +351,21 @@ export const test = {
     })
     assert.equal(spaceInfoResult.out.error, undefined)
 
-    assert.deepEqual(confirmationEmail.to, email, 'emails are equal')
+    assert.deepEqual(confirmationEmail.to, accountEmail, 'emails are equal')
   },
   'authorize with poll claim': async (assert, context) => {
-    const { email, account, agent, service, mail } = await setup({
+    const { accountEmail, account, agent, service, mail } = await setup({
       ...context,
-      email: 'example@dag.house',
+      accountEmail: 'example@dag.house',
     })
 
-    const authorized = authorizeAndWait(agent, email, {
+    const authorized = authorizeAndWait(agent, accountEmail, {
       expectAuthorization: waitForAuthorizationByPolling,
     })
+    // wait 100 milliseconds to let the email be sent
+    await new Promise((resolve) => {
```

Comment:
`mail.take()` should take care of waiting until mail is there, so this seems obsolete